.travis.yml

```diff
@@ -15,6 +20,25 @@ jobs:
     - stage: jdk11 with FIPS
       script: lein with-profile fips test
       jdk: openjdk11
+
+    - stage: jdk8 without FIPS
+      script: lein with-profile dev test
+      jdk: openjdk8
+      arch: ppc64le
+    - stage: jdk8 with FIPS
```

Comment:
The FIPS cell has erred twice now, I'm not sure why, but it could very well be an issue with our internal build system. FIPS is only supported in our downstream enterprise product, but, like you, we want to catch issues that might foul it up early so we test it here. You can probably exclude those cells from using ppc64le if you like.